src/staging/bot.py

```diff
@@ -5,18 +5,22 @@
 import string
 import tempfile
 import xml.etree.ElementTree as ET
-from collections.abc import Callable
-from collections.abc import Coroutine
-from collections.abc import Generator
-from collections.abc import Iterable
 from dataclasses import dataclass
 from dataclasses import field
 from datetime import datetime
 from datetime import timedelta
 from functools import reduce
+from io import BytesIO
+from pathlib import Path
+from typing import Callable
 from typing import ClassVar
+from typing import Coroutine
```

Comment:
why? https://docs.python.org/3/library/typing.html#typing.Coroutine and https://peps.python.org/pep-0585/